### DISCUSSION TITLE
Feature request: Support asset fetching via ros-foxglove-bridge that have `file://` url

### DISCUSSION THREAD

**Discussion by HovorunBh** [foxglove/community#1018] [General]
I see that `file://` format is supported by [ros-foxglove-bridge](https://github.com/foxglove/ros-foxglove-bridge/pull/232) but studio will only send request to ros-foxglove-bridge for asset fetching if the url is `package://` See https://github.com/foxglove/studio/pull/6229.

Would be nice to make it also support `file://` urls :)

**Comment by banisadr** [comment]
@HovorunBh can you elaborate on what you're trying to do and what workflows are/aren't working for you?

**Comment by achim-k** [comment]
Internal tracking issue: ERT-981

**Reply by achim-k** [reply]
This has been implemented and is already available in the web app. It will be available in the desktop app with the next release.

If you use Foxglove Bridge, you may also need to update its asset_uri_allowlist parameter to permit WebSocket clients to fetch file:// URLs.
